src/hp/magicsock/conn.rs

```diff
@@ -181,6 +181,8 @@ pub struct Inner {
     closing: AtomicBool,
     /// Close was called.
     closed: AtomicBool,
+    /// Are we currently processing STUN responses (set during a netcheck report).
+    enable_stun_packets: AtomicBool,
```

Comment:
I wonder if we should replace this by a method on the `netcheck::Client` instead of tracking this here.  It seems like a better place to track whether the netcheck client is running or not.

It's a bit difficult though, `netcheck::Client` isn't `Clone` right now, maybe we should make it `Clone`, I think it would simplify things here.

Reply:
Can we just always forward stun packets and let the receiver decide whether they are needed?